integration/flink/app/src/main/java/io/openlineage/flink/visitor/VisitorFactoryProvider.java

```diff
@@ -0,0 +1,51 @@
+/*
+/* Copyright 2018-2024 contributors to the OpenLineage project
+/* SPDX-License-Identifier: Apache-2.0
+*/
+
+package io.openlineage.flink.visitor;
+
+import org.apache.flink.FlinkVersion;
+
+public class VisitorFactoryProvider {
```

Comment:
This class can be removed for now.

Reply:
If unused yet, I'd remove it and add when needed